libraries/mongodb.rb

```diff
@@ -22,6 +22,35 @@
 require 'json'
 
 class Chef::ResourceDefinitionList::MongoDB
+  def self.create_replicaset_member(node)
```

Comment:
Which documentation style should I use?
I was trying to rdoc but the docs are really lackluster, if you could point me out to a good source that would be of great help :)